### ISSUE TITLE
Modify the read lilypond sequence

### ISSUE THREAD

**Issue by scaredycode** [UHHRobotics22-23/marimbabot#184]
- [ ] increase/decrease tempo by fixed value
- [ ] increase/decrease tempo by specified value
- [ ] set tempo to specified value
- [ ] increase/decrease volume by fixed value
- [ ] increase/decrease volume by specified value
- [ ] set volume to specified value
- [ ] add swing
- [ ] transpose
- [ ] ...

**Comment by Juphex** [comment]
Are the other ones in progress @scaredycode ?

**Comment by ateRstones** [comment]
#212 seems to be included in this ticket? I'm wondering @scaredycode

**Comment by scaredycode** [comment]
> Are the other ones in progress @scaredycode ?

no, @yunlong-wang-cn and I decided against including anymore lilypond modification functionality.
I tested adding swing and transpositions some weeks ago, but this of course caused the parser to fail :)

Regarding setting volume to a specified value, I suggested adding commands such as "Play in fortissimo" or similar, but it is currently not implemented neither in the speech node nor in the behavior. We also thought that this might not be a use case a "usual" user would go for (more "Play x steps louder" might be much more intuitive).

**Comment by Juphex** [comment]
@scaredycode 
I close this because you moved this into "Done".
